proposals/2020/Add Resource Use Notes.md

```diff
@@ -0,0 +1,31 @@
+# Add ResourceUseNotes element
+
+## Overview
+
+This proposal is to add the `auc:ResourceUseNotes` child element to the `auc:ResourceUse` element.
+
+## Justification
+
+Standard 211 sections 6.1.2.1.d and 6.1.2.1.j specify
+> ... If meter sampling is used by the qualified energy auditor, [the report must include] the sampling methodology used for each fuel
+> ... Identification of irregularities in monthly energy use patterns and suggestions about their possible causes
+
+This proposal suggests that having an element freeform text notes within `auc:ResourceUse` would best address these requirements.
+
```

Comment:
Add something along the lines:

"This is similar to the approach taken for freeform text necessary to be defined at the `auc:Building` level, where the `auc:PremisesNotes` field is used to capture freeform text inputs from users.